recipe/meta.yaml

```diff
@@ -11,23 +11,27 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 1
+  number: 0
   script: {{ PYTHON }} -m pip install . --no-deps -vv
-  skip: true  # [py<37 or win or osx]
+  skip: true  # [python_impl == 'pypy']
+  skip: true  # [py<38 or win or osx]
 
 requirements:
   host:
     - python
     - pip
   run:
     - python
+    - databroker =1.2
```

Comment:
This will restrict our deployment using tiled/databroker v2. What's the issue with them?

Also, why does hkl depend on databroker?

Reply:
How is tiled using hklpy?

Reply:
[databroker is used](https://github.com/search?q=repo%3Abluesky%2Fhklpy%20databroker&type=code) by documentation and testing to support recovery of crystal orientation from run(s) in a catalog.  This code was added in this release.  Refactoring and a new release of hklpy would be needed to remove the pin.

Can we accept this release as-is with commitment that the pin will be removed in a new version soon?  The databroker2 environment will build, but it would solve to hklpy v1.0.3, which does not have this pin.

Reply:
As it's a dev/doc requirement, I would remove databroker from the dependencies (we don't need docs generation in conda, and that pinned version can be optionally added to the tests block if needed). It is not needed to _run_ the code.

Reply:
Let's try it.

Reply:
Zzza!  Works for me.